src/components/Footer.jsx

```diff
@@ -42,22 +42,42 @@ const Footer = () => {
 
             <div className="flex flex-row mt-2 align-end">
               <FaInstagram className="mr-2" />
-              <p className="font-semibold text-lg">@ucrgardening</p>
+              <a
```

Comment:
footer changes shouldn't be in this branch, this is being addressed in your other Pull Request. Remove all changes in this file